java/src/main/java/com/codurance/training/tasks/refactored/service/TaskService.java

```diff
@@ -0,0 +1,17 @@
+package com.codurance.training.tasks.refactored.service;
+
+
+
+import com.codurance.training.tasks.refactored.Task;
+
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Map;
+
+public interface TaskService {
+    void addTask(String project, String description,String taskId, Map<String, List<Task>> tasks, PrintWriter out);
+
+    void deleteTask(String taskId, Map<String, List<Task>> tasks, PrintWriter out);
```

Comment:
delete method should be moved to a seperate interface since the class which implements this interface will be having multiple reponsibilities which violates SRP